new notebook: notification for task completion

xiaosi-xu opened the pull request:
*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

lundql reviewed (changes requested):
Very nice work! I tightened up some things, broke up a few long sentences, implemented a few specific items of style guidance, and generally aimed toward reducing the mental overhead required from the reader. That is a general instruction for all AWS writing, even though we know our customers are very intelligent---they are also very busy!

xiaosi-xu commented:
> Very nice work! I tightened up some things, broke up a few long sentences, implemented a few specific items of style guidance, and generally aimed toward reducing the mental overhead required from the reader. That is a general instruction for all AWS writing, even though we know our customers are very intelligent---they are also very busy!

Thank you for the review! Your comments are quite useful. I have incorporated all the changes and made a new commit.

lundql reviewed (approved):
LGTM
